working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.                if value_counter &gt; 0:
</pre><p>Are you sure this is <em>really</em> working? :)</p><p> </p><p>Besides, same consideration about post conditions as reported below! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>